app/src/main/java/com/spoony/spoony/presentation/auth/onboarding/OnboardingState.kt

```diff
@@ -0,0 +1,20 @@
+package com.spoony.spoony.presentation.auth.onboarding
+
+import com.spoony.spoony.core.designsystem.component.textfield.NicknameTextFieldState
+import com.spoony.spoony.core.designsystem.model.RegionModel
+
+enum class OnboardingSteps(val step: Float) {
+    ONE(1f),
+    TWO(2f),
+    THREE(3f),
+    END(0f)
+}
```

Comment:
p2
step을 직접 적어주면 추후 값이 늘어날 때마다 적어줘야 할게 많아져용~

```suggestion
enum class OnboardingSteps() {
    END,
    ONE,
    TWO,
    THREE;
    
    val step: Float
        get() = this.ordinal.toFloat()
}
```

요건 어떠신지?

Reply:
만약에 step이 늘어났을 때 어떤 뷰가 추가될지 몰라서(progressBar 유무 등등,,) 저렇게 넣어줬는데 어차피 뷰 추가되면 저거 다 수정해야해서😅 현재 상황에서는 오빠가 말해준대로 하는게 더 나을수도 있을 것 같아요!! 감사합니둥~